tests/wallet/test_singleton.py

```diff
@@ -19,13 +21,13 @@ def test_only_odd_coins():
         [did_core_hash, did_core_hash, 1, [0xDEADBEEF, 0xCAFEF00D, 210], 205, [[51, 0xCAFEF00D, 205]]]
     )
     try:
-        result, cost = DID_CORE_MOD.run_with_cost(INFINITE_COST, solution)
+        cost, result = SINGLETON_MOD.run_with_cost(INFINITE_COST, solution)
```

Comment:
Use `assertRaises` to be more explicit about what you're testing for. See https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaises